.github/workflows/kustomize-gke-destroy.yaml

```diff
@@ -0,0 +1,67 @@
+name: Helm GKE Destroy
+# Reusable workflow for destroying a Helm chart on GKE Kubernetes
+
+on:
+  workflow_call:
+    inputs:
+      release-name:
```

Comment:
I think this should be called `kustomization-path` as well since you are not referring to a release in the cluster here